.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: csharp
+script:

```

Comment:
In my head I thought it mentioned we needed a `solution: solution-name.sln` as well?